commit 75e36da1ae129f721c4dc3bbb3368dec8d294611
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Wed Jul 4 17:01:15 2018 +0100

    staging: bcm2835-camera: Remove check of the number of buffers supplied
    
    Before 9384167 there was a need to ensure that there were sufficient
    buffers supplied from the user to cover those being sent to the VPU
    (always 1).
    With 9384167 the buffers are linked 1:1 between MMAL and V4L2,
    therefore there is no need for that check, and indeed it is wrong
    as there is no need to submit all the buffers before starting streaming.
    
    Fixes: 9384167 "staging: bcm2835-camera: Remove V4L2/MMAL buffer remapping"
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/bcm2835-camera/mmal-vchiq.c b/drivers/staging/vc04_services/bcm2835-camera/mmal-vchiq.c
index 6c643cbde0fc..9a0c25b68ac2 100644
--- a/drivers/staging/vc04_services/bcm2835-camera/mmal-vchiq.c
+++ b/drivers/staging/vc04_services/bcm2835-camera/mmal-vchiq.c
@@ -1338,16 +1338,6 @@ static int port_enable(struct vchiq_mmal_instance *instance,
 	if (port->enabled)
 		return 0;
 
-	/* ensure there are enough buffers queued to cover the buffer headers */
-	if (port->buffer_cb) {
-		hdr_count = 0;
-		list_for_each(buf_head, &port->buffers) {
-			hdr_count++;
-		}
-		if (hdr_count < port->current_buffer.num)
-			return -ENOSPC;
-	}
-
 	ret = port_action_port(instance, port,
 			       MMAL_MSG_PORT_ACTION_TYPE_ENABLE);
 	if (ret)
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1402_75e36da1ae129f721c4dc3bbb3368dec8d294611.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1402_75e36da1ae129f721c4dc3bbb3368dec8d294611.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
